packages/design-system/src/components/pill/stories/index.js

```diff
@@ -76,21 +107,27 @@ const PILL_OPTIONS = [
   { id: 4, label: 'John' },
 ];
 
-function PillGroupContainer() {
+function PillGroupContainer(args) {
   const [active, setActive] = useState(1);
   return (
     <Container>
-      <PillGroup options={PILL_OPTIONS} value={active} onSelect={setActive} />
+      <PillGroup
+        options={PILL_OPTIONS}
+        value={active}
+        onSelect={setActive}
+        {...args}
+      />
     </Container>
   );
 }
 
 // Override light theme because this component is only set up for dark theme right now given fg and bg coloring
-export const PillGroups = () => (
+export const PillGroups = (args) => (
   <>
     <PillGroupContainer />
```

Comment:
💭 This one needs to have `args` passed in too

![image](https://user-images.githubusercontent.com/22185279/151243260-4fac71d2-bcf0-4e5e-9fd4-8544466152f7.png)

Reply:
This one didn't actually have any functioning `controls` set up so the props are filtered out. We don't want to show controls for pills if they are used. Here we are letting the user set the selected state by clicking the `pill` as opposed to setting it in a `control`. Don't know why it was setup that way. Just copied it over.